.circleci/config.yml

```diff
@@ -249,6 +249,16 @@ jobs:
             # skip swagger validation because of difficult-to-fix operationId duplications
             # java -jar openapi-generator-cli.jar validate -i dockstore-webservice/src/main/resources/swagger.yaml
             java -jar openapi-generator-cli.jar validate -i dockstore-webservice/src/main/resources/openapi3/openapi.yaml
+      - run:
+          name: Check Java imports sorted
+          # A dirty Java file after a build probably means it was committed without building, and its imports are out of order
+          command: |
+            if [[ $(git diff --name-only | grep "\.java$") != '' ]]; then
```

Comment:
To my surprise, although it makes sense when you think about it, there were other files showing up as modified -- the decrypted migrations files. So, look for modified Java files only.

Reply:
I forget, do we have a separate check for the openapi.yaml?
Does it make sense to put these two together?

Reply:
Good catch, I added the check for openapi.yaml.

Reply:
To be clear, I  don't remember the story behind this (i.e. I would have though we would have had it somewhere else before, but maybe we removed it), but worth a try